lib/bolt/cli.rb

```diff
@@ -451,6 +456,10 @@ def file_stat(path)
       File.stat(path)
     end
 
+    def outputter
```

Comment:
This function seems unnecessary and confusing when we're only going to use the outputter in a single function lifetime.